.github/workflows/POAM.yml

```diff
@@ -0,0 +1,52 @@
+name: Weekly Team Sync
+on:
+  schedule:
+    - cron: 0 * 1 1-12 *
+
+jobs:
+  create_issue:
+    name: Create monthly POAM issue
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: Create monthly POAM issue
+        run: |
+          if [[ $CLOSE_PREVIOUS == true ]]; then
+            previous_issue_number=$(gh issue list \
+              --label "$LABELS" \
+              --json number \
+              --jq '.[0].number')
+            if [[ -n $previous_issue_number ]]; then
+              gh issue close "$previous_issue_number"
+              gh issue unpin "$previous_issue_number"
+            fi
+          fi
+          new_issue_url=$(gh issue create \
+            --title "$TITLE" \
+            --assignee "$ASSIGNEES" \
+            --label "$LABELS" \
+            --body "$BODY")
+          if [[ $PINNED == true ]]; then
+            gh issue pin "$new_issue_url"
+          fi
```

Comment:
What's this doing?

Reply:
This gives us the option to close the previous versions of this issue when the new one is created. I don't think we'd use it right now, but I left it in case we liked the automation and wanted to try it down the line. We would just switch `CLOSE_PREVIOUS` at the bottom of the page to `true`.

We would need to clean up the labels to make it more specific to these POAM issues. Probably just by adding a POAM label or something like that.

Reply:
Happy to remove if we want to prune the action a bit since it's not currently being used!

Reply:
Yes, while useful in the future — it'd be better to start with a simpler version and see how it works for us.

Reply:
Removed in [9e3434f](https://github.com/uswds/uswds-team/pull/366/commits/9e3434ffb3b538197ada13931db2f57819f88873)